src/frontend-samples/fire-decoration-sample/README.md

```diff
@@ -0,0 +1,30 @@
+# Particle Effect - Fire
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This sample show how to create a particle effect using fire as an example.
+
+## Purpose
+
+The purpose of this sample is to demonstrate the following:
+
+* Creating a decorator and rendering it in the active view.
+* Working with particle effects using the [ParticleCollectionBuilder](https://www.itwinjs.org/reference/imodeljs-frontend/rendering/particlecollectionbuilder) API.
+* The visualizing (NOT simulating) fire in an iModel.
+
+## Description
+
+This sample provides an example of a particle effect in world space (as apposed to view space).  The sample provides a tool to place and controls limitedly configure fire particle emitters.  Each emitter is initially configured based with preset parameters defined in [FireDecorationApp.ts](./FireDecorationApp.tsx).  Select which preset is use with the drop down menu.  Once you have deselected an emitter, there is no way to return to configuring it.
```

Comment:
/s/apposed/opposed
Remove "controls limitedly".
/s/based with/based on
/s/is use/is used

Reply:
/s/apposed/opposed